tools_iuc.yaml

```diff
@@ -1635,6 +1635,10 @@ tools:
     owner: devteam
     tool_panel_section_label: Phenotype Association
 
+  - name:  suite_glimmer
```

Comment:
Pleas do not install suite but the tools individual y. Suite don't work really.

Reply:
I was in doubt indeed